### PR TITLE
Add parameter for logging JSON traffic to stdout

### LINKED ISSUE #116: Allow logging of enigma-go session traffic
For debugging purposes, allowing our end-users to dump the websocket traffic to stdout would be useful.

E.g.

```bash
corectl --log-traffic build
```

### PR DESCRIPTION
**Pull request by wennmo**
This PR adds a parameter `--traffic` or `-t` that enables JSON traffic logging to `stdout`.

Closes #116

### CODE REVIEW

File: docs/corectl.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ corectl [flags]
 
 ```
   -h, --help      help for corectl
+  -t, --traffic   Log JSON traffic to stdout
```
Comment (axelssonHakan) [inline docs/corectl.md]: QIX WS traffic?

Comment (wennmo) [reply docs/corectl.md]: In my opinion we shouldn't use `QIX`, but maybe `Log JSON websocket traffic to stdout`?

File: main.go [hunk 2]
```diff
@@ -966,6 +966,9 @@ func init() {
 	corectlCommand.PersistentFlags().BoolP("verbose", "v", false, "Logs extra information")
 	viper.BindPFlag("verbose", corectlCommand.PersistentFlags().Lookup("verbose"))
 
+	corectlCommand.PersistentFlags().BoolP("traffic", "t", false, "Log JSON traffic to stdout")
```
Comment (FredrikFolkesson) [inline main.go]: change to "Log JSON websocket traffic to stdout" as above :)

File: internal/log.go [hunk 3]
```diff
@@ -11,3 +11,19 @@ func LogVerbose(message string) {
 		fmt.Println(message)
 	}
 }
+
+var LogTraffic bool
+
+type TrafficLogger struct{}
+
+func (l *TrafficLogger) Opened() {}
```
Comment (FredrikFolkesson) [inline internal/log.go]: Remove the object, and test if it works with a value receiver instead of a pointer

**Pull request merged**